src/python/pants/backend/jvm/tasks/scalafix.py

```diff
@@ -5,14 +5,15 @@
 from abc import abstractmethod
 
 from pants.backend.jvm.tasks.rewrite_base import RewriteBase
+from pants.backend.jvm.tasks.jvm_task import JvmTask
 from pants.base.exceptions import TaskError
 from pants.java.jar.jar_dependency import JarDependency
 from pants.option.custom_types import file_option
 from pants.task.fmt_task_mixin import FmtTaskMixin
 from pants.task.lint_task_mixin import LintTaskMixin
 
 
-class ScalaFix(RewriteBase):
+class ScalaFix(RewriteBase, JvmTask):
```

Comment:
This should not be necessary, because `RewriteBase` already extends `NailgunTask`, which extends `JvmTask`.

Reply:
`NailgunTask` extends `JvmToolTaskMixin` which extends `JvmToolMixin`... which doesn't extend `JvmTask`?

Also: `Exception message: 'ScalaFixGoal_scalafix' object has no attribute 'classpath'` (albeit from a slightly older version)

Reply:
Got it.